green_cli/authenticators/__init__.py

```diff
@@ -140,6 +140,7 @@ def default_hw_device_info(self):
                   'name': self.name,
                   'supports_low_r': False,
                   'supports_liquid': 0,
+                  'supports_host_unblinding': True,
```

Comment:
Not sure this is the correct place for this flag.  
May instead want it set in WallyAuthenticatorLiquid (ie wally.py ~ 160)

Reply:
Agreed, updated.